test/operations_test.cpp

```diff
@@ -1754,6 +1763,26 @@ namespace
     BOOST_TEST(time_diff >= -60.0 && time_diff <= 60.0);
   }
 
+  //  creation_time_tests  ----------------------------------------------------------------//
+
+  void creation_time_tests(const fs::path& dirx)
+  {
+    cout << "creation_time_tests..." << endl;
+    fs::path f1x = dirx / "foobar2";
+    create_file(f1x, "foobar2");
+
+# ifdef BOOST_POSIX_API
+#  ifdef __linux__
+#    if LINUX_VERSION_CODE < KERNEL_VERSION(4,11,0)
+    BOOST_TEST(CHECK_EXCEPTION(bad_creation_time, ENOSYS));
+#    endif
+#  endif
+# else
+    std::time_t ft = fs::last_write_time(f1x);
+    BOOST_TEST(ft == fs::creation_time(f1x));
+# endif
```

Comment:
This doesn't run the test on POSIX systems. Just run the tests unconditionally and return if you get `ENOSYS`.

Reply:
I fix it.